include/ck/tensor_operation/gpu/device/device_conv3d_fwd_xdl_ndhwc_kzyxc_ndhwk.hpp

```diff
@@ -345,8 +301,7 @@ struct DeviceConv3dFwdXdl_Input_N_Di_Hi_Wi_C_Weight_K_Z_Y_X_C_Output_N_Do_Ho_Wo_
 
     using CGridDesc_M0_N0_M1_N1_M2_M3_M4_N2 =
         decltype(GridwiseGemm::MakeCGridDescriptor_M0_N0_M1_N1_M2_M3_M4_N2(CGridDesc_M_N{}));
-    using Block2CTileMap =
-        decltype(Block2CTileMapMaker{1}.MakeBlock2CTileMap(CGridDesc_M_N{}, 1, 1));
+    using Block2CTileMap = typename GridwiseGemm::DefaultBlock2CTileMap;
```

Comment:
It doesn't seem correct. This device_conv3d is using a customized mapper, which works with ```kernel_gemm_xdlops_v2r3_for_conv3d``` together that convert conv3d to a batched-gemm and then to a gemm.

I'm surprised to see this change passed the verification

Reply:
This custom mapper is the same for batched gemm. Batched gemm and gemm can actually use the same mapper so I blanket-apply it to every gridwise gemm except for split-k gemm.

Reply:
makes sense, thanks for explaining